pkg/util/apiserver/cors_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package apiserver
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/verrazzano/verrazzano-operator/pkg/api/instance"
+)
+
+var allowedHTTPMethods = []string{"POST", "GET", "OPTIONS", "PUT", "DELETE"}
+var allowedHeaders = []string{"Accept", "Content-Type", "Content-Length", "Accept-Encoding", "X-CSRF-Token", "Authorization"}
+
+//TestEnableCors tests that the EnableCors method adds the expected Access-Control-Allow-Origin header to the response
+func TestEnableCors(t *testing.T) {
+	type args struct {
+		verrazzanoURI string
+		writer        http.ResponseWriter
+	}
+	tests := []struct {
+		name                string
+		origin              string
+		expectedAllowOrigin string
+		args                args
+	}{
+		{"With Verrazzano URI set", "https://console.somesuffix.xip.io",
```

Comment:
would be nice to have a test with a port number on the end, just to see what happens

Reply:
added a case for that